src/dbnode/storage/bootstrap/types.go

```diff
@@ -473,3 +474,22 @@ type NamespaceDetails struct {
 	// Shards are the shards to retrieve info files for in the specified namespace.
 	Shards []uint32
 }
+
+// SeriesRef is used to both write to and load blocks into a database series.
+type SeriesRef interface {
```

Comment:
should `DatabaseSeries` be updated to embed this interface?

Reply:
It could... I opted not to since `SeriesRef` is a bootstrap only iface in addition to having to do some shuffling around of type defs to avoid a circular import.